### PR TITLE
Add url to sourceview docs

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Linked to https://github.com/gtk-rs/docs/pull/4

### CODE REVIEW

**Comment by EPashkin** [comment]
Will remove unused footnote?

**Comment by GuillaumeGomez** [comment]
Yup. :)

File: docs-src/index.md [hunk 1]
```diff
@@ -35,5 +36,3 @@ users will have it installed.
 ### Examples
 
 See the [examples repository](https://github.com/gtk-rs/examples).
-
-### [The GTK+ Project documentation](http://www.gtk.org/documentation.php)
```
Comment (EPashkin) [inline docs-src/index.md]: Seems is wrong deletion

Comment (GuillaumeGomez) [inline docs-src/index.md]: Hum no, try on the website. This url shouldn't exist.

Comment (EPashkin) [inline docs-src/index.md]: It works for me

Comment (GuillaumeGomez) [inline docs-src/index.md]: Woups nevermind...

**Comment by EPashkin** [comment]
👍

**Pull request merged**